home/serializers.py

```diff
@@ -198,6 +199,19 @@ def body_field_representation(page, request):
     ):
         if page.whatsapp_body != []:
             try:
+                # if it's a template, we need to get the template content
+                if template := page.whatsapp_template_or_none:
+                    return OrderedDict(
+                        [
+                            ("text", template.message),
+                            ("is_whatsapp_template", "True"),
+                            ("whatsapp_template_name", template.name),
+                            (
+                                "whatsapp_template_category",
+                                template.category,
+                            ),
+                        ]
+                    )
```

Comment:
Does this mean that, if it is a template, then we won't return all the other page fields? (like `message`, `next_message`, etc). Maybe this should just set variables `text`, `is_whatsapp_template`, and `whatsapp_template_name` (which are set differently if it's not a template), and then we return the same full body at the end.

Also, how does this work when they request `message=2`? Won't this then just return the same template for every message, instead of returning either a template or a message depending on which message they selected?